### DISCUSSION TITLE
Task file inputs - how to reference a file from the monorepo root?

### DISCUSSION THREAD

**Discussion by ValentinGurkov** [vercel/turborepo#7514] [Help]
### Summary

Hi everyone,

I come from a monorepo setup where the is an `nx` configuration at the monorepo root. For a task, there are variables such as `{workspaceRoot}` that can be used to refer to a file relative from the project root.

I am trying to achieve something similar with `turbo`, is it possible?

Since the monorepo packages have a different degree of nesting, I can't simply use a relative path to the file.

### Additional information

I want to replicate something like this `nx` config with `trubo`
```js

{
    "lint": {
        "inputs": ["default",  "{projectRoot}/.eslintrc.cjs", "{workspaceRoot}/common/eslint-config/index.cjs"],
        "cache": true
    }
}
```

**Comment by mehulkar** [comment]
Paths are all relative from the working directory of the package. You would have to traverse with `../../` etc

**Reply by ValentinGurkov** [reply]
Yeah., though the path may be for one package like `../../` and for another `../../../`. due to nesting. Can't think of anything besides all possible relative paths for this file

**Reply by shinebayar-g** [reply]
Could you support variables like `workspaceRoot` & `projectRoot`? They make inputs & outputs configuration soo much easier.
